d3a_api_client/utils.py

```diff
@@ -268,3 +272,18 @@ def log_bid_offer_confirmation(message):
                          f"{round(energy, 2)} kWh at {price} cts/kWh")
     except Exception as e:
         logging.error(f"Logging bid/offer info failed.{e}")
+
+
+def get_simulation_info(simulation_id=simulation_id_from_env, domain_name=domain_name_from_env,
```

Comment:
Please add a small unittest for this function. Thanks!

Reply:
https://github.com/gridsingularity/d3a-api-client/pull/94/commits/302c1dc2d64d90d29244a6376e97da49878838bc